### PR TITLE
Update teams-store-validation-guidelines.md

### PR DESCRIPTION
**Pull request by ramendhe**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1a0409ccf9559491864b260f71d8834b4f363278/msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md) | 87 | 75 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e1cedca-e2b5-45ed-b88e-333fa028e9df) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1a0409ccf9559491864b260f71d8834b4f363278/msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md) | 87 | 75 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1f302e26-eb1c-4091-af67-11c6d93180a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ If your app includes a tab, make sure it adheres to these guidelines.
 * Tabs in a personal context may aggregate content from shared instances of the app.
 * Tabs must be responsive to Teams themes. When a user changes the theme, the app's theme must reflect the selection.
 * Tabs must use Teams-styled components, such as, Teams fonts, type ramps, color palettes, grid system, motion, tone of voice, and so on whenever possible.
-* You must include a **Settings** tab.
+* Include a 'Settings' tabs if your app has functionality that requires a changes in the settings.
```
Comment (v-abirade) [inline msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md:300]: Please rephrase:
```suggestion
* If your app functionality requires changes in the settings, include a **Settings** tab.
```

**Review by v-abirade — CHANGES_REQUESTED** [review]
Please rephrase.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa142d303abd2dd45b9cd9abe01a440d27acde9f/msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md) | 87 | 74 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5f912d0-5981-4aa9-a4d5-0afb7455e39c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f5d81790fc5a9aa5c4f08af115f6cecf7aada510/msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md) | 87 | 75 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52e804bb-2694-4bcb-9e4b-d05c7f7f669c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**